### PR TITLE
Editorial correction related to wrong usage of registrar terminology

### PR DESCRIPTION
**Pull request by cdh4u**

### CODE REVIEW

File: draft-ietf-sipcore-sip-push.xml [hunk 1]
```diff
@@ -466,23 +473,18 @@ align="center"><artwork>
...
+          In case of non-2xx response to the REGISTER request, and the proxy has not yet forwarded the SIP request towards the UA, the proxy MUST reject the SIP request with a 404 (Not Found) response. 
         </t>
         <t>
           If the push notification request fails (see PNS-specific documentation for details), the proxy MUST reject the SIP request with a 
           556 (Push Notification Failed) response.
         </t>
-        <t>
-          NOTE: As described above, the reason the proxy needs to wait for the REGISTER response before forwarding the SIP request is to 
-          make sure that the REGISTER request has been accepted by the registrar, and that the registered contact matches the Request-URI 
-          of the SIP request to be forwarded.
```
Comment (rshpount) [inline draft-ietf-sipcore-sip-push.xml]: We do not care if the registered contact matches the Request-URI. What we care about is that REGISTER message came from a party authorized to receive requests for this contact.

Comment (cdh4u) [reply draft-ietf-sipcore-sip-push.xml]: Yes, but we need to be able to match the SIP request to be forwarded with the correct REGISTER request. There could be multiple REGISTER requests, from different UAs.

Comment (rshpount) [reply draft-ietf-sipcore-sip-push.xml]: What I meant was proxy can validate that registered contact matches the Request-URI without waiting for REGISTER response.

What I think you are trying to say is something like: 
The reason the proxy needs to wait  for the REGISTER response before forwarding the SIP request is to make sure that the REGISTER request has been accepted by the registrar, and that the UA which initiated the REGISTER request is authorized to receive messages for the Request-URI.

Comment (cdh4u) [reply draft-ietf-sipcore-sip-push.xml]: I have submitted a new commit which fixes that. Note that the Note your are commenting on has actually been removed, and the text has been moved up a few paragraphs.

**Comment by rshpount** [comment]
The latest text looks good.

**Pull request merged**